src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -39,13 +42,15 @@ public SharingService(AuthMiddleware auth,
                           KeysetStoreWriter storeWriter,
                           RotatingKeysetProvider keysetProvider,
                           IKeysetKeyManager keyManager,
+                          KeysetManager keysetManager,
```

Comment:
KeysetManager is supposed to wrap RotatingKeysetProvider, KeysetStoreWriter, IKeysetKeyManager
but now we're passing all 4 - is that needed?

Reply:
Yes until we move everything over

Reply:
why do we still need KeysetStoreWriter and IKeysetKeyManager?

Reply:
I have created another ticket that covers the rest of the migration to the Keyset Manager, I think its too big for 1 ticket.